third_party/silabs/cmake/utility.cmake

```diff
@@ -32,9 +32,9 @@ string(TOUPPER "${EFR32_PLATFORM}" PLATFORM_UPPERCASE)
 string(TOLOWER "${BOARD}" BOARD_LOWERCASE)
 string(TOUPPER "${BOARD}" BOARD_UPPERCASE)
 
-set(SILABS_GSDK_DIR ${PROJECT_SOURCE_DIR}/third_party/silabs/sdk_support)
+set(SILABS_GSDK_DIR ${PROJECT_SOURCE_DIR}/third_party/silabs/gecko_sdk)
 
 # Check if GSDK exists
 if(NOT EXISTS "${SILABS_GSDK_DIR}")
-    message(FATAL_ERROR "Cannot find: ${SILABS_GSDK_DIR}\nCheck that Silicon Labs GSDK ${SILABS_GSDK_VERSION} was installed properly")
+    message(FATAL_ERROR "Cannot find: ${SILABS_GSDK_DIR}\nPlease make sure the 'third_party/silabs/gecko_sdk' submodule is initialized")
```

Comment:
Good point. I actually just removed it since we already print out the path

<img width="738" alt="Screen Shot 2022-06-01 at 4 36 01 PM" src="https://user-images.githubusercontent.com/2881693/171497073-4978c9d3-f7b8-4385-8af1-3d7a9abbbf39.png">